docs/email/components.html

```diff
@@ -0,0 +1,528 @@
+            </tr>
+            <tr class="fs-caption">
+                <td><code class="stacks-code">color</code> (background color)</td>
+                <td>VML in <code class="stacks-code">&lt;v:fill&gt;</code></td>
+                <td>Always set to make foreground HTML legible if background image doesn't load.</td>
+            </tr>
+        </tbody>
+    </table>
+    <div class="stacks-preview">
+{% highlight html linenos %}
+<td valign="middle" style="background-image: url('https://d2axdqolvqmdvx.cloudfront.net/857c3f94-f791-401e-85a8-e041ec72569e/bgheropatternblue.png'); background-position: center center; background-size: cover; background-color: #117FF1; text-align: center;">
+    <!--[if gte mso 9]>
+    <v:rect xmlns:v="urn:schemas-microsoft-com:vml" fill="true" stroke="false" style="width:680px;height:220px;">
+    <v:fill type="tile" src="https://d2axdqolvqmdvx.cloudfront.net/857c3f94-f791-401e-85a8-e041ec72569e/bgheropatternblue.png" color="#117FF1"/>
+    <v:textbox inset="0,0,0,0">
+    <![endif]-->
+    <div>
+        <!-- Foreground Content : BEGIN -->
+        <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation">
+            <tr>
+                <td style="padding: 60px; color: #ffffff; font-family: arial, sans-serif; font-size: 15px; text-align: center;">
+                    Foreground HTML content.
+                </td>
+            </tr>
+        </table>
+        <!-- Foreground Content : END -->
+    </div>
+    <!--[if gte mso 9]>
+    </v:textbox>
+    </v:rect>
+    <![endif]-->
+</td>
+{% endhighlight %}
+        <div class="stacks-preview--example">
+            <table border="0" cellpadding="0" cellspacing="0" width="100%" role="presentation">
+                <tr>
+                    <td valign="middle" style="background-image: url('https://d2axdqolvqmdvx.cloudfront.net/857c3f94-f791-401e-85a8-e041ec72569e/bgheropatternblue.png'); background-position: center center; background-size: cover; background-color: #117FF1; text-align: center;">
+                        <div>
+                            <!-- Foreground Content : BEGIN -->
+                            <table align="center" border="0" cellpadding="0" cellspacing="0" role="presentation">
+                                <tr>
+                                    <td style="padding: 60px; color: #ffffff; font-family: arial, sans-serif; font-size: 15px; text-align: center;">
+                                        Foreground HTML content.
+                                    </td>
+                                </tr>
+                            </table>
+                            <!-- Foreground Content : END -->
+                        </div>
+                    </td>
+                </tr>
+            </table>
+        </div>
+        <aside class="s-notice s-notice__info s-notice__outlined mt24" role="status" aria-hidden="false">
```

Comment:
Move notice outside of `div.stacks-preview--example`. Getting a connected box-shadow effect.

![image](https://user-images.githubusercontent.com/1544281/43609835-08b8d3f4-9673-11e8-8f01-338ad8cee39f.png)